AirCasting/APICommunicator/APICommunication.swift

```diff
@@ -0,0 +1,31 @@
+// Created by Lunar on 01/07/2021.
+//
+
+import Foundation
+
+final class APICommunication {
```

Comment:
Let's change it to something like `ForgotPasswordAPI` (and filename too) to match codebase conventions.